server/db/models/cartCandy.js

```diff
@@ -3,25 +3,31 @@ const db = require('../db')
 const {Cart, Candy} = require('../models')
 
 const CartCandy = db.define('cartCandy', {
-  // BLOCKER: waiting on candy model for foreign keys -- H
-  // candyId: {
-  //   type: Sequelize.INTEGER,
-  //   allowNull: false,
-  //   references: {
-  //     model: Candy,
-  //     key: 'id'
-  //   }
-  // },
-  // cartId: {
-  //   type: Sequelize.INTEGER,
-  //   allowNull: false,
-  //   references: {
-  //     model: Cart,
-  //     key: 'id'
-  //   }
-  // },
+  candyId: {
```

Comment:
arent candyId and cartId added automatically when you make the associations?. I though this model would only have "amount".

Reply:
I'm not sure if it's necessary to have these foreign key fields in the model definition, since the association is established in the db/index.js file. It doesn't appear to be interfering the with models' functionality, but it might be superfluous -- let's ask @dpatlut when we get a chance, but move forward for now since I need this new definition to test other model instance methods. <3